src/org/javarosa/core/io/Std.java

```diff
@@ -0,0 +1,32 @@
+package org.javarosa.core.io;
+
+import java.io.PrintStream;
+
+/**
+ * Used to break dependency directly on System.out and System.err.
+ */
+public class Std {
+
+    public static PrintStream out = System.out;
```

Comment:
> Do you have an approach for a quick sanity check?
So, after implementing a similar one myself, I realized that the console output in briefcase is split between a modest amount output from javarosa and similar, but far more verbose amount from similar `System.out` code in briefcase that prints warnings pulled from the jr API. Yours test is probably working, but just not easy to see with all the other console output.